### DISCUSSION TITLE
Teeth segmentation

### DISCUSSION THREAD

**Discussion by kirrog** [Project-MONAI/MONAI#5896] [Q&A]
Hello everybody!

We are trying to solve teeth segmentation problem with monai.
We tried to segment teeth one by one, but it's not robust and any bias break our model results. And of course it takes to much time to learn every model.
We find vertebra segmentation approach, by Andreas(@diazandr3s - https://github.com/Project-MONAI/MONAILabel/discussions/1225#discussioncomment-4476370) advice. Does anyone have experience with this model to apply it to teeth segmentation?

**Comment by diazandr3s** [comment]
Hi @kirrog,

You may find this project interesting: https://projectweek.na-mic.org/PW38_2023_GranCanaria/Projects/TeethSegmentation/
Do you have access to a dataset? @dgmato may be able to provide more details of the MONAI Label system he's using for dental applications.

**Comment by dgmato** [comment]
@kirrog we're using a multi-stage segmentation approach for this task. In a first stage we localize the teeth and in a second stage we segment the individual teeth. This multi-stage approach works much better than "deepedit" or "segmentation" sample apps. We are using the vertebra pipeline as a reference for this.

**Comment by abhishekpatil32** [comment]
Hello @diazandr3s are there any pre-trained models available for use for teeth segmentation using CT scans???

I am interested in using the auto3dseg for the segmentation of the teeth.

**Reply by diazandr3s** [reply]
Hi @abhishekpatil32,

I'm sorry for the late response.
I'm not sure we have an Auto3DSeg model for teeth segmentation. I know @dgmato has trained a multi-stage system for teeth segmentation. Here is the infer script: https://github.com/dgmato/MONAILabel/blob/dental-pipeline/sample-apps/radiology/lib/infers/dental_pipeline.py
Also, there are folks working on this task for the Project Week happening in Jan 2025. Maybe you could join the preparation meetings and ask more info? https://projectweek.na-mic.org/PW42_2025_GranCanaria/

Hope this helps,

**Comment by okkarkyaw122** [comment]
Thank you for the initiative. Are there anyone trying to segment enamel and dentine separately?
